api/CSSKeyframeRule.json

```diff
@@ -28,31 +43,81 @@
             },
             {
               "version_added": "5",
-              "prefix": "moz"
+              "version_removed": "48",
+              "prefix": "Moz"
             }
           ],
           "ie": {
             "version_added": "10"
           },
-          "opera": {
-            "version_added": "12",
-            "prefix": "o"
```

Comment:
This did seem to be wrong, there's no `OCSSKeyframeRule` or `oOCSSKeyframeRule` in Opera 12.16.